conferences/abstracts/OSF19_Demo_submission.md

```diff
@@ -0,0 +1,91 @@
+# OSFair2019 Demo Submission 
+Open Science Fair 2019, September 16-18, Porto, Portugal
```

Comment:
Can you add details in the bullet points as in https://github.com/alan-turing-institute/the-turing-way/blob/master/conferences/abstracts/ukoswg_submission.md#uk-oswg-uk-network-of-open-science-working-groups ?